### PR TITLE
Create folder with exemplar - create minimal.cdl

### PR DESCRIPTION
**Pull request by vturpin**
Here I propose the minimal .cdl

I've removed everything that is not mandatory.
This is align with the status of the format the 14th of February 2024 with downgrading of PARAMETER and SENSOR informations.

Impact should be low as the agreement on what is mandatory in OG.1.0 is pretty weak. But a final check is necessary before release.

Proponents: 
Moderator: @OceanGlidersCommunity/format-mantainers

# Type of PR

- [ ] Typo without possible change of interpretation of the related text.
- [ ] Fix of some error, inconsistency, unforeseen limitation.
- [ ] Style that only affects visually the compiled document
- [X] Addition that does not require change in the current structure.
- [ ] Enhancement that require changes to improve the format.

# Related Issues

# Dates when it got review approvals

# Release checklist

- [ ] Approved by at least two members of the committee?
- [ ] There were modifications after the review approvals? If so, please
      ask reviewers to update their review.
- [ ] Proponents and moderador should explicitly agree that it is ready to
      to merge.
- [ ] The moderador is the one in charge to actually merge or close this PR
      according to the final decision.

# For maintainers

- [ ] Update the moderator with a volunteer from the committee. It would be
      best to have one single moderator to guide and help this PR to move
      forward. It is OK to update the moderador pass it to another one.
- [ ] Confirm that the associated branch was deleted after the merging.
- [ ] Wrap-up and close the related issues.

# Comments

### CODE REVIEW

File: examples/minimal.cdl [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+            TIME_GPS:valid_max = 4e9 ;
+            TIME_GPS:ancillary_variables = "TIME_GPS_QC" ;
+        double LATITUDE(N_MEASUREMENTS) ;
+            LATITUDE:long_name = "latitude of each measurement and GPS location" ;
+            LATITUDE:standard_name = "latitude" ;
+            LATITUDE:units = "degrees_north" ;
+            LATITUDE:_FillValue = -9999.9 ;
+            LATITUDE:valid_min = "-90" ;
+            LATITUDE:valid_max = "90" ;
+            LATITUDE:ancillary_variables = "LATITUDE_GPS_QC"
```
Comment (callumrollo) [inline examples/minimal.cdl:36]: missing a ; at the end of this line

File: examples/minimal.cdl [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+            :rtqc_method_doi = "10.21105/joss.02063" ;
+            :date_created = "2021-04-10T01:14:44" ;
+            :featureType = "trajectory" ;
+            :Conventions = "CF-1.8, ACDD-1.3, OG-1.0" ;
+
+    data:
+    
+     TRAJECTORY = "sp041_20191205T1757" ;
+    
+     PARAMETER = "PRES", "TEMP", "PSAL", "CHLA", "DOXY" ;
```
Comment (callumrollo) [inline examples/minimal.cdl:158]: PARAMETER hasn't been defined in the variables section

File: examples/minimal.cdl [hunk 3]
```diff
@@ -0,0 +1,220 @@
+netcdf sp041_20191205T1757 {
```
Comment (callumrollo) [inline examples/minimal.cdl:1]: As per OG conventions, I think the filename should be something like sp041_20191205T1757_delayed.nc

**Review by callumrollo — CHANGES_REQUESTED** [review]

**Review by vturpin — COMMENTED** [review]
I'll update to adjust the file to the 3 remarks.

File: examples/minimal.cdl (hunk 1 above)
Comment (vturpin) [reply examples/minimal.cdl]: changed made in the file (plus " ;" where space was missing)

**Review by glidermann — APPROVED** [review]

**Pull request merged**